scripts/helper.js

```diff
@@ -99,6 +99,16 @@ function tryOverwrite(f) {
             if (ow.delete) {
                 delete references[k];
                 console.log("Deleted", k);
+            } else if (ow.renameTo) {
+                references[ow.renameTo] = ref;
+                delete references[k];
+                console.log("Renamed", k, "to", ow.renameTo);
+
+                if (ow.aliasOf && ow.aliasOf.replaceWith) {
+                    console.log("Adding", k, "...");
+                    console.log("   ", "aliasOf:", undefined, "->", JSON.stringify(ow.aliasOf.replaceWith));
+                    references[k] = { aliasOf: ow.aliasOf.replaceWith };
+                }
```

Comment:
Could this be more succinctly and clearly expressed with something like `switchWith` where you'd write:

```json
{
 "hr-time": {
        "switchWith": "hr-time-1",
    },
    "hr-time-1": {
        "href": { "replaceWith": "https://www.w3.org/TR/hr-time-1/" }
    }
}
```

It seems this is an common enough need that we should cater to it specifically.

Reply:
Except&nbsp;that `hr‑time` is&nbsp;an&nbsp;alias for&nbsp;`hr‑time‑2`, not&nbsp;`hr‑time‑1`.

Reply:
Oh! That's a good point. 

I'm still uneasy with what's going on, here.